pkg/master/master.go

```diff
@@ -138,6 +137,21 @@ type Config struct {
 	MasterCount int
 }
 
+type Config struct {
+	GenericConfig *genericapiserver.Config
+	ExtraConfig
```

Comment:
I really dislike anonymous inclusion.  I'm ok with the extra layer, but I'm not a fan of doing this.  Is it required to make the `completedConfig` work as expected?

Reply:
No, it was only convenience on my side. Will make it explicit.